team_access_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
```

Comment:
https://pkg.go.dev/github.com/stretchr/testify/assert is more active/popular

https://pkg.go.dev/github.com/google/go-cmp/cmp is good too